terraform/modules/stack/iam_role_policy.tf

```diff
@@ -188,29 +188,6 @@ resource "aws_iam_role_policy" "allow_azure_verifier_access_cache" {
   policy = data.aws_iam_policy_document.azure_verifier_tags_readwrite.json
 }
 
-# backfill policies
-# TODO: delete everything under this comment once the azure migration is done
-
-resource "aws_iam_role_policy" "bags_tracker_vhs_manifests_backfill_readonly" {
-  role   = module.bags_tracker_backfill.task_role_name
-  policy = var.vhs_manifests_readwrite_policy_backfill
-}
-
-resource "aws_iam_role_policy" "bags_tracker_backfill_metrics" {
-  role   = module.bags_tracker_backfill.task_role_name
-  policy = data.aws_iam_policy_document.cloudwatch_putmetrics.json
-}
-
-resource "aws_iam_role_policy" "bag_register_backfill_replica_primary_readonly" {
-  role   = module.bag_register_backfill.task_role_name
-  policy = data.aws_iam_policy_document.replica_primary_readonly.json
-}
-
-resource "aws_iam_role_policy" "bag_register_backfill_metrics" {
-  role   = module.bag_register_backfill.task_role_name
-  policy = data.aws_iam_policy_document.cloudwatch_putmetrics.json
-}
-
 resource "aws_iam_role_policy" "azure_replicator_replica_primary_readonly" {
```

Comment:
I think we can delete this policy too – the Azure replicator should be reading from the unpacker bucket, not the primary storage.